hebiros_gazebo_plugin/include/hebiros_gazebo_controller.cpp

```diff
@@ -331,13 +331,38 @@ double HebirosGazeboController::ComputeForce(std::shared_ptr<HebirosGazeboGroup>
 
   gear_ratio = hebiros_joint->gear_ratio;
 
+  float voltage = 48.0;
```

Comment:
fyi, this requires narrowing from double to float. If it's a float, I _always_ make sure to use `48.0f`, etc